src/basic/hashmap.c

```diff
@@ -705,7 +705,7 @@ static unsigned hashmap_iterate_entry(HashmapBase *h, Iterator *i) {
                                                : hashmap_iterate_in_internal_order(h, i);
 }
 
-bool internal_hashmap_iterate(HashmapBase *h, Iterator *i, void **value, const void **key) {
+bool _hashmap_iterate(HashmapBase *h, Iterator *i, void **value, const void **key) {
```

Comment:
not too much of a fan of such bikeshedding excercsies, but ok

Reply:
hmm, now e have `hashmap_iterate()` and `_hashmap_iterate()`, that's a bit confusing, no? one one character difference, and that character is kinda a whitespace that isn't a whitespace...

Reply:
I think it's pretty clear which one is the "internal" function and which one is supposed to be used. This convention is widely used in Python and I don't think mistaking one for the other is common in any way.